languages.toml

```diff
@@ -1713,3 +1713,13 @@ language-server = { command = "pasls", args = [] }
 [[grammar]]
 name = "pascal"
 source = { git = "https://github.com/Isopod/tree-sitter-pascal", rev = "2fd40f477d3e2794af152618ccfac8d92eb72a66" }
+
+[[language]]
+name = "gdscript"
+scope = "source.gdscript"
+injection-regex = "gdscript"
+file-types = ["gd", "gdscript", "gdscript3", "gdscript4"]
+comment-token = "#"
+indent = { tab-width = 2, unit = "	" }
+language-server = { command = "nc", args = ["localhost", "6008"] }
+# Change this to port 6005 once Godot 4 is released.
```

Comment:
This seems to duplicate an existing grammar definition. Just edit the one on line 1321